region/norfair/east.json

```diff
@@ -4883,10 +4883,10 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                    "SpaceJump",
```

Comment:
I would keep space jump on the base strat, then add a second strat which includes wall jumps and gravity.

Reply:
There are no walljumps, only movement efficiency.   I found spacejump to be misleading since its usage is unlike in ampitheater which has a similar room structure.  You save time by optimizing run speed and keeping it going with space.  And that's not worth 60 heat frames to explain imo.

Reply:
Oh. I didn't realize this was going L -> R, unlike the rest of the strats. Never mind this all looks good!